### ISSUE TITLE
Support for Linux builds of Unity

### ISSUE THREAD

**Issue by rodhall85** [aevitas/unity-patch#8]
Firstly, great work on this! 

Your readme.md implies that this should work for Linux builds but I can't get mine to work on any Linux build, I have tried `2019.1.14f1`, `2019.2.11f1` and `2019.3.0b10` (beta build) and in each case I recieve  

```
Opening Unity executable from /home/rod/unity/2019.1.14f1/Editor/Unity...
Creating backup..
Backup file created.. looks awake.
Searching for theme offset..
Could not find the theme offset in the specified executable.
```

I'm looking through your code and I can only see patterns for Windows and OSX. Am I doing something wrong? (I might be!) here is the command I am running: 

```
sudo ./linux-x64/Patcher -e=/home/rod/unity/2019.1.14f1/Editor/Unity -t=dark
```

If you know what to search for in the Linux builds, please let me know, I am happy to do some digging, I have done it manually using a hex editor before (on Windows and MacOS) and I will update and send in a PR if I can get it to work.

Thanks
Rod

**Comment by aevitas** [comment]
Hey Rod, thanks for getting in touch!

You are right the readme wrongly suggested the patcher supports Linux, and I've (hopefully) fixed that now by adding support for Linux to the patcher: https://github.com/aevitas/unity-patch/releases/tag/v1.03

And I've updated the readme to properly indicate the `-linux` flag required for Linux builds of Unity3D.

The version I pulled the pattern from is 2019.2.3f, but I doubt it'll change much between minor versions.

Let me know if this works for you!

**Comment by rodhall85** [comment]
Hey. Thanks for taking the time to update for me. It says that it's worked! However... when I open Unity, it seems not to have worked :frowning: I wonder if the Unity Hub is interfering with it (although it's fine for when I'm in Windows)

![image](https://user-images.githubusercontent.com/19157112/68618844-2807fc80-04c2-11ea-82c0-e357c090642e.png)

Any ideas? 

What tool are you using to disassemble the binary? (assuming that's what you are doing?) I've had success with x64dbg in Windows.

**Comment by rodhall85** [comment]
I've got it :smiley: 

As far as I can see, there is a bug in your code. I can't actually see how it works for any version right now :see_no_evil: 

```[c#]
byte themeByte = themeName == "dark" ? (byte)0x74 : (byte)0x75;
```

I think these should be the other way around: 

```[c#]
byte themeByte = themeName == "dark" ? (byte)0x75 : (byte)0x74;
```

From what I can see in the rest of the code, this will work for all OS's but can't confirm. I have tested this using the `-t=light` switch and it works, DARK MODE IS ON :smile: 

Thanks mate for your work on this, really appreciated!
